modules/management/kubectl-apply/variables.tf

```diff
@@ -41,15 +41,15 @@ variable "kueue" {
   description = "Install and configure [Kueue](https://kueue.sigs.k8s.io/docs/overview/) workload scheduler. A configuration yaml/template file can be provided with config_path to be applied right after kueue installation. If a template file provided, its variables can be set to config_template_vars."
   type = object({
     install              = optional(bool, false)
-    version              = optional(string, "v0.8.1")
+    version              = optional(string, "v0.9.0")
     config_path          = optional(string, null)
     config_template_vars = optional(map(any), null)
   })
   default = {}
 
   validation {
-    condition     = !var.kueue.install || contains(["v0.8.1"], var.kueue.version)
-    error_message = "Supported version of Kueue is v0.8.1"
+    condition     = !var.kueue.install || contains(["v0.9.0", "v0.8.1"], var.kueue.version)
```

Comment:
Store this in a list, and use it in the error message as well as the condition

Reply:
Requesting clarification: Isn't ["v0.9.0", "v0.8.1"] already a list? Are you recommending that we should store it in a new variable and then use it elsewhere?

Reply:
Yes, use a new variable and avoid the hardcoded versions in the error message. Use the variable in the error message